cardano-node-chairman/test/Test/Cardano/Node/Chairman.hs

```diff
@@ -55,13 +58,26 @@ prop_spawnOneNode = H.propertyOnce . H.workspace "temp/chairman" $ \tempDir -> d
     , "--secret-seed", "2718281828"
     ]
 
+  H.writeFile (tempDir <> "/genesis/GENHASH") . L.unlines . L.reverse . L.take 1 . L.reverse . L.lines =<< H.execCli
+    [ "print-genesis-hash"
+    , "--genesis-json"
+    , tempDir <> "/genesis/genesis.json"
+    ]
```

Comment:
This is what the shell scripts that start the cluster does.  I think the `GENHASH` is necessary piece of cluster configuration.